src/ControlzEx.Showcase/Themes/Controls.xaml

```diff
@@ -419,10 +417,11 @@
                                     </Grid>
                                 </Grid>
                                 <Popup x:Name="Popup"
-                                       controlzex:PopupBackdropManager.BackdropType="AcrylicBlurbehind"
+                                       RenderOptions.ClearTypeHint="Enabled"
+                                       TextOptions.TextFormattingMode="Display"
                                        MinWidth="{TemplateBinding ActualWidth}"
                                        VerticalAlignment="Center"
-                                       AllowsTransparency="True"
+                                       AllowsTransparency="False"
```

Comment:
This was meant as a showcase for the acrylic effect.
How should we showcase those now?

Reply:
I don't know, maybe also per option and use then a different style.

Reply:
Sounds like a good idea.